config/routes.rb

```diff
@@ -2,7 +2,7 @@
   #post 'followships/create'
   #post 'followships/destroy'
   resources:followships,only: [:create,:destroy,:update]
-
+  resources:notifications,only:[:index,:show,:destroy]
   #patch '/followships/accept' => 'followships#accept'
   #resources :comments
```

Comment:
How many time do I have to tell you. 
**Remove Commented code if not needed before publishing your work**

Reply:
Done